spec/models/season/phase_switcher_spec.rb

```diff
@@ -1,35 +1,43 @@
 require 'spec_helper'
 
+describe Season::PhaseSwitcher do
 
-context 'when switching phases' do
-  subject { Season.current }
+  it "is available as described class" do
```

Comment:
Since this is now sorted out, please remove this again, no need to test rspec's internals.